src/Native/Runtime/CachedInterfaceDispatch.cpp

```diff
@@ -86,7 +86,7 @@ static void * UpdatePointerPairAtomically(void * pPairLocation,
                                           void * pSecondPointer,
                                           bool fFailOnNonNull)
 {
-#if defined(_X86_) || defined(_ARM_)
+#if !defined BIT64
```

Comment:
I would swap the order of if and else blocks, and make the condition positive. Positive conditions are easier to read.

Reply:
Done